bin/propolis-server/Cargo.toml

```diff
@@ -47,7 +47,8 @@ slog = "2.7"
 propolis = { path = "../../lib/propolis", features = ["crucible-full", "oximeter"], default-features = false }
 propolis-client = { path = "../../lib/propolis-client" }
 propolis-server-config = { path = "../../crates/propolis-server-config" }
-rfb = { git = "https://github.com/oxidecomputer/rfb", branch = "main" }
+#rfb = { git = "https://github.com/oxidecomputer/rfb", branch = "main" }
```

Comment:
note: will remove these (and other) local rfb deps when the associated rfb PR (https://github.com/oxidecomputer/rfb/pull/8) is merged